cli/config_file.go

```diff
@@ -155,7 +158,9 @@ func SettingsFileReadWrite(state *Settings) *Settings {
 
 	// Settings Validation steps
 
-	// TODO(c3ppc3pp): Check for available ports
+	if settings.Command == "up" {
```

Comment:
Even there was the comment for checking, it was misleading sadly. (Whoops, sorry) 

I think it would be better to place this in the runner.go file, inside ProcessCommand() function, so it will be a last step before start. I'm personally not sold on the idea of we are being able to write these port numbers back to the settings file. @nandor-magyar What about you?

Reply:
moved to ProcessCommand() before docker container object init

Reply:
Perfect! Thanks!